athena-postgresql/src/main/java/com/amazonaws/athena/connectors/postgresql/PostGreSqlQueryStringBuilder.java

```diff
@@ -42,9 +47,13 @@
 public class PostGreSqlQueryStringBuilder
         extends JdbcSplitQueryBuilder
 {
-    public PostGreSqlQueryStringBuilder(final String quoteCharacters, final FederationExpressionParser federationExpressionParser)
+    private final java.util.Map<String, String> configOptions;
+    private final String postgresqlCollateExperimentalFlag = "postgresql_collate_experimental_flag";
```

Comment:
I assume this name will change once it is not experimental?

Reply:
I'll answer offline